doc/getting-started/getting-started.rst

```diff
@@ -328,11 +328,11 @@ Generate a Board Configuration File
          disk="/media/$USER/"$(ls /media/$USER)
          cp -r "$disk"/board_inspector ~/acrn-work
 
-#. On the target, load the ``msr`` driver, used by the Board Inspector:
+#. On the target, update the ``pci.ids`` file which contains the list of known identifiers related to PCI devices:
```

Comment:
Looks like the best choice now is to remove this step completely (lines 331-336) and instead update the board_inspector.py when it calls ``lspci`` to add the ``-q`` option (to fetch missing IDs).